Makefile

```diff
@@ -368,5 +376,5 @@ kind-delete-hosted: $(ADDON_CONTROLLER)
 .PHONY: 
 kind-deploy-cert-manager:
 	@echo installing cert-manager on managed
-	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.12.0/cert-manager.yaml --kubeconfig=$(PWD)/kubeconfig_$(MANAGED_CLUSTER_NAME)
+	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/${CERT_MANAGER_VERSION}/cert-manager.yaml --kubeconfig=$(PWD)/kubeconfig_$(MANAGED_CLUSTER_NAME)
```

Comment:
Good catch! I didn't even know this was down here!

Reply:
hehe Thanks!!